config/switchport.py

```diff
@@ -0,0 +1,119 @@
+import click
+from .utils import log
+import utilities_common.cli as clicommon
+
+# since default vlan id is 1
+default_vid = 1
+
+#
+# 'switchport' mode ('config switchport ...')
+#
+
+
+@click.group(cls=clicommon.AbbreviationGroup, name='switchport')
+def switchport():
+    """Switchport mode configuration tasks"""
+    pass
+
+
+@switchport.add_command("mode")
+@click.argument("type", metavar="<mode_type>", Required=True, type=click.Choice(["access", "trunk", "routed"]))
+@click.argument("port", metavar="port", required=True)
+@clicommon.pass_db
+def access_mode(db, type, port):
```

Comment:
Why cant we enhance the existing commands (sudo config vlan member...) to add additional validations? FYI. we do config validations in the YANG models now, please explore the same.